lib/ClangImporter/SwiftBridging/swift/bridging

```diff
@@ -71,10 +83,10 @@
 ///    object.doSomething()
 ///    // The Swift compiler will release object here.
 ///  ```
-#define SWIFT_SHARED_REFERENCE(_retain, _release)                                \
-  __attribute__((swift_attr("import_reference")))                          \
-  __attribute__((swift_attr(_CXX_INTEROP_STRINGIFY(retain:_retain))))      \
-  __attribute__((swift_attr(_CXX_INTEROP_STRINGIFY(release:_release))))
+#define _CXX_INTEROP_GET_MACRO(_1, _2, _3, NAME, ...) NAME
+#define SWIFT_SHARED_REFERENCE(...)                                            \
```

Comment:
Is there a way to define this as `(_retain, _release, ...)`? That would make it more readable for someone opening this header file.